CHANGELOG.md

```diff
@@ -77,6 +77,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.4.0] - 2020-06-11
 ### Added
 - Index to PII and Non-PII database collections. [#428](https://github.com/rokwire/rokwire-building-blocks-api/issues/428)
+- Add search option in contribution and capability dataset. [#458](https://github.com/rokwire/rokwire-building-blocks-api/issues/458)
```

Comment:
After https://github.com/rokwire/rokwire-building-blocks-api/pull/571 gets reviewed and merged, please move the entries in this PR for Contributions catalog under the `[Unreleased]` section. Thanks.

Reply:
updated

Reply:
Thanks.